### PR TITLE
Add: add array.empty

### PR DESCRIPTION
**Pull request by Jim8y**
closes https://github.com/neo-project/neo-devpack-dotnet/issues/861

### CODE REVIEW

File: src/Neo.Compiler.CSharp/MethodConvert/MethodConvert.cs [hunk 1]
```diff
@@ -430,6 +430,11 @@ private void ConvertExtern()
                             break;
                     }
                 }
+                if ((bool)Symbol.ToString()?.StartsWith("Array.Empty"))
```
Comment (Jim8y) [inline src/Neo.Compiler.CSharp/MethodConvert/MethodConvert.cs:433]:
```suggestion
                if ((bool)Symbol.ToString()?.StartsWith("System.Array.Empty"))
```

File: tests/Neo.Compiler.CSharp.UnitTests/TestClasses/Contract_Array.cs [hunk 2]
```diff
@@ -89,6 +90,12 @@ public static object TestStructArray()
             return sarray[2];
         }
 
+        public static object[] TestEmptyArray()
```
Comment (shargon) [inline tests/Neo.Compiler.CSharp.UnitTests/TestClasses/Contract_Array.cs:93]: It was not executed this test?

Comment (Jim8y) [reply tests/Neo.Compiler.CSharp.UnitTests/TestClasses/Contract_Array.cs:93]: I forgot to commit the UT

**Comment by Jim8y** [comment]
Good to go now. @shargon

**Review by superboyiii — APPROVED** [review]
Works well on mine.

**Pull request merged**